RecoTauTag/RecoTau/src/AntiElectronIDMVA6.cc

```diff
@@ -11,6 +11,10 @@
 #include "CondFormats/DataRecord/interface/GBRWrapperRcd.h"
 #include "FWCore/Framework/interface/ESHandle.h"
 
+#include "MagneticField/Engine/interface/MagneticField.h"
+#include "MagneticField/Records/interface/IdealMagneticFieldRecord.h"
+
+#include <TMath.h>
```

Comment:
Hi David, 

I'm not sure I understand what you are referring to, therefore I'll answer to both interpretations that I'm having: 

(1)
The MagneticFiled headers are needed for the track extrapolation we are doing in case that the variable "phiAtEcalEntrace" has not been pre-calculated to be saved in miniAOD (the Bfield is used in function atECalEntrance).  

(2)
TMath actually is not used any more as it seem to me. If you want me to do it I could delete the line and recommit, while I'm a bit hesitant to start the whole signing and testing procedure again on this long thread. 

Cheers,
Roger